### DISCUSSION TITLE
API Dash v0.3.0

### DISCUSSION THREAD

**Discussion by animator** [foss42/apidash#88] [Announcements]
Our biggest release of API Dash ⚡️ is now available 🚀 with lots of amazing new features.

Key highlights of this release are:
🗂️ Support for collections and folders to easily organize APIs.
🖥 Well tested code generators for cURL, HAR, Python (`requests`, `http.client`), JavaScript (`axios`, `fetch`), node.js (`axios`, `fetch`) & Kotlin (`okhttp3`).
{..} Interactive JSON Response viewer with a Find Box to easily explore the results.
🔤 Auto-suggestions for the most popularly used headers.
🔵 Tab indicators for Request URL Parameters, Headers and Body tabs to quickly identify if they are populated.
🧑‍🎨 Updated UI & Settings.
📄 Support for PDF response.
🎵 Support for Audio (wav , mp3) response.
🏃‍♂️ Support for APNG response.
↗️ Export collections & folders into a HAR file that can be version controlled & can be directly imported in other API Clients. You can also export your entire data via Settings.
❓ Updated Help & Support page.

Check out the  [CHANGELOG](https://github.com/foss42/apidash/blob/main/CHANGELOG.md) for image preview of all the above highlighted features.

A big thank you to these wonderful developers for their contributions in this release: [@aqsasayyed](https://github.com/aqsasayyed), [@mmjsmohit](https://github.com/mmjsmohit), [@Dushant-Bansal](https://github.com/Dushant-Bansal), [@Mixel2004](https://github.com/Mixel2004), [@morpheus-30](https://github.com/morpheus-30), [@madhupashish](https://github.com/madhupashish) & [@DenserMeerkat](https://github.com/DenserMeerkat).

All binaries for Windows, MacOS and Linux are provided below👇
We would definitely love to hear your feedback 🙏
Do support this project by giving it a star ⭐️ and sharing it with your friends 🤗

Installation instructions are available [here](https://github.com/foss42/apidash/blob/main/INSTALLATION.md).

<hr /><em>This discussion was created from the release <a href='https://github.com/foss42/apidash/releases/tag/v0.3.0'>API Dash v0.3.0</a>.</em>

**Comment by animator** [comment]
Shoutout to the following developers for their contributions in this release: @aqsasayyed, @mmjsmohit, @Dushant-Bansal, @Mixel2004, @morpheus-30, @madhupashish & @DenserMeerkat 🎉
Thank you folks!

**Reply by madhupashish** [reply]
sir This year i want this as my first project of GSOC
